team-agreements/definition-of-ready/readme.md

```diff
@@ -16,6 +16,8 @@ It can be understood as a checklist that helps the Product Owner to ensure that
 * [ ] Does the user story have clear and complete acceptance criteria?
 * [ ] Does the user story address the business need?
 * [ ] Can we measure the acceptance criteria?
+* [ ] Can the story be broken down into smaller stories?
+* [ ] Is the story blocked?

```

Comment:
I think it would be worth providing some examples of blocked scenarios. This mostly comes up with dependencies/predecessors. For example, this story cant be completed until this other story is finished. Another common case is we are waiting on some sort of decision external to the team before we can complete (i.e. theres an outstanding question that needs to be answered by LT or another team.)